codecov.yml

```diff
@@ -9,7 +9,7 @@ coverage:
     patch:
       default:
         target: 80%
-        informational: true
+        informational: false
```

Comment:
I don't think we should do this - blocking patches on code coverage can be overly restrictive, especially for smaller changes, and can put a big burden on authors in places where good testing doesn't already exist.

We'd also need buy in from partner teams since this would impact code that our core team doesn't own.

Reply:
Note that right now our repo-level coverage is ~40% and its unclear to me if that includes integration tests